ga4gh/datamodel/reads.py

```diff
@@ -432,17 +433,31 @@ class SimulatedReadGroup(AbstractReadGroup):
     """
     def __init__(self, parentContainer, localId, randomSeed, numAlignments=2):
         super(SimulatedReadGroup, self).__init__(parentContainer, localId)
+        self._randomSeed = randomSeed
 
     def getReadAlignments(self, referenceId=None, start=None, end=None):
+        rng = random.Random(self._randomSeed)
+        if start is not None:
+            # Skip ahead to start id
+            for i in xrange(start):
```

Comment:
A more efficient approach might be to have 
```
rng.seed(self._randomSeed + start)
```
this way we avoid having to iterate over (potentially) a very large range.